spec/requests/error_handling_spec.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
-RSpec.describe ErrorsController, type: :controller do
+RSpec.describe ErrorsController, type: :request do
   describe 'GET #not_found' do
     it 'returns http 404' do
-      get :not_found
+      get '/errors/not_found'
       expect(response).to have_http_status(404)
     end
   end
 
   describe 'GET #internal_server_error' do
     it 'returns http 500' do
-      get :internal_server_error
+      get '/errors/internal_server_error'
```

Comment:
```suggestion
      get errors_internal_server_error_path
```

Nitpick: Same here with Rails route helpers.